src/platforms/common/performance/instrumentation/performance-metrics.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Performance Metrics
+sidebar_order: 20
+supported:
+  - javascript
+notSupported:
+  - javascript.cordova
+  - javascript.electron
+  - react-native
+  - dotnet
+  - python
+  - go
+  - java
+  - android
+  - ruby
+  - java.spring-boot
+  - apple
+  - unity
+  - dart
+  - flutter
+  - rust
+  - native
+  - php
+  - native.breakpad
+  - native.crashpad
+  - native.minidumps
+  - unreal
+  - native.wasm
+description: "Learn how to attach performance metrics to your transactions."
+---
+
+Sentry's SDKs support sending performance metrics data to Sentry. These are numeric values attached to transactions that are aggregated and displayed in Sentry.
+
+<PlatformContent includePath="performance/automatic-performance-metrics" />
+
+In addition to automatic performance metrics, the SDK supports setting custom performance metrics on transactions. This allows you to define metrics that are important to your application and send them to Sentry.
+
+To set a performance metric, you need to supply the following:
+- name (`string`)
+- value (any numeric type - `float`, `integer`, etc.)
+- unit (`string`, can be empty or the string `'none'`). 
+
+Sentry supports adding arbitrary custom units, but we recommend using one of the [supported units listed below](./#supported-performance-metric-units).
+
+<PlatformContent includePath="performance/custom-performance-metrics" />
+
+## Supported Performance Metric Units
```

Comment:
Adding this at the behest of @edwardgou-sentry.

```suggestion
<Alert level="warning" title="Note">

For the moment, unit conversion is only supported once the data has already been stored. This means that, for example, `('myMetric', 60, 'second')` and `('myMetric', 3, 'minute')` would not be aggregated, but rather stored as two separate metrics. To avoid this, make sure to use a consistent unit when recording a custom metric.

</Alert>

## Supported Performance Metric Units
```